depends/README_USAGE.md

```diff
@@ -0,0 +1,38 @@
+To build dependencies for the current arch+OS:
+
+    make
+
+To build for another arch/OS:
+
+	make HOST=host-platform-triplet && make HOST=host-platform-triplet
```

Comment:
So just change to:

     To build for another arch/OS:
          make HOST=host-platform-triplet